tools/make_dictionary_file.py

```diff
@@ -0,0 +1,66 @@
+"""
+Makes dictionary file from mozc
+(https://github.com/google/mozc/tree/master/src/data/dictionary_oss)
+"""
+
+def check_number(id1, id2, numbers):
+    """
+    checks id1 == id2 == numbers
+    """
+    if id1 != id2:
+        return False
+
+    if id1 in numbers and id2 in numbers:
+        return id1 == id2
```

Comment:
if id1 is in numbers and id1 == id2, then id2 is in numbers, so you don't need to check "id2 in numbers" ;)